docs/products/flink/index.rst

```diff
@@ -0,0 +1,59 @@
+Aiven for Flink
+===============
+
+What is Aiven for Flink?
+------------------------
+
+Aiven for Flink is a fully managed **distributed processing engine for stateful computations over data streams**, deployable in the cloud of your choice enabling the creation of streaming data pipelines on top of your datasets.
+
+
+Why Flink?
+-----------------
+
+Flink is a processing engine enabling the definition of streaming data pipelines with SQL statements. It can work in batch or streaming mode, it's distributed by default and performs computation at in-memory speed at any scale.
+
+
+Get started with Aiven for Flink
+--------------------------------
+
+Take your first steps with Aiven for Flink by following our :doc:`getting-started` article, or browse through our full list of articles:
+
+
+.. panels::
+
+    📙 :doc:`concepts`
+
+    ---
+
+    💻 :doc:`howto`
+
+
+Integrates with your existing tools
+------------------------------------
+
+Flink is highly compatible with other Aiven products for the following tasks:
+
+- Parse, transform, calculate streaming data from Aiven for Kafka.
+
+- Ingest changes on relational PostgreSQL or MySQL tables.
+
+- Join data coming from various sources listed above to create complex data pipelines.
+  
+- Stream the data pipeline result back to Apache Kafka or any of the supported sinks.
+
+
+
+
+Ways to use Aiven for Flink
```

Comment:
Remove this empty section please. If we need it, create a ticket to add it later when we have the content.

Reply:
Removed